extern/storage-sealing/states_replica_update.go

```diff
@@ -59,10 +59,6 @@ func (m *Sealing) handleSubmitReplicaUpdate(ctx statemachine.Context, sector Sec
 		return nil
 	}
 
-	if err := checkPieces(ctx.Context(), m.maddr, sector, m.Api); err != nil { // Sanity check state
```

Comment:
double checking that you intended to remove this because it's not described in the commit

Reply:
Yes, I did: the subsequent call to checkReplicaUpdate calls checkPieces as the first thing it does